deploy/036_deploy_blast_spokepool.ts

```diff
@@ -10,7 +10,7 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   const initArgs = [
     // Initialize deposit counter to very high number of deposits to avoid duplicate deposit ID's
     // with deprecated spoke pool.
-    1_000_000,
+    2_000_000,
```

Comment:
this shouldonyl be changed when we update the spoke pool address. That's because this spoke pool proxy's `depositId` state variable won't change if the implementation contract changes

Reply:
Done: https://github.com/across-protocol/contracts/pull/565/commits/408ce045b073b555283659650990295c435be5af